eosdatadistribution/producers/usb.py

```diff
@@ -56,7 +56,7 @@ def mount_removed_cb(monitor, mount, store):
     pprint.pprint([store.remove_name(n) for p, n in store.producers.items() if p.startswith(p)])
 
 
-if __name__ == '__main__':
+def main():
```

Comment:
is that needed ? seems to work fine without it, and i feel it makes the code easier to read not to add another brain dereference.